src/js/views/forms/form/form.scss

```diff
@@ -146,3 +146,40 @@
     }
   }
 }
+
+.form__save-button-container {
+  display: flex;
+}
+
+.form__save-button {
+  border-bottom-right-radius: 0;
+  border-top-right-radius: 0;
+  margin-right: 1px;
+}
+
+.form__save-dropdown-button {
+  border-radius: 0 4px 4px 0;
+  padding: 0;
+  width: 24px;
+
+  .icon {
+    font-size: 12px;
+    vertical-align: -0.125em;
+    width: 1em;
+  }
+}
+
+.form__save-dropdown-picklist-item {
+  align-items: center;
+  display: flex;
+}
+
+.form__save-dropdown-picklist-icon {
+  margin-left: auto;
+
+  .icon {
+    font-size: 12px;
+    margin-right: 0 !important;
```

Comment:
I don't like using `!important`, but it's the only way I could get this property applied to the `.icon` here.

It's being overridden by the Picklist component's styling: [.picklist__item .icon:first-child](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/components/picklist/picklist.scss#L53).

Without `margin-right: 0` here, the icon would not have the correct spacing:

![Screen Shot 2022-09-26 at 4 24 22 PM](https://user-images.githubusercontent.com/35355575/192383435-7e4aa760-e31a-413f-8b1c-0865a9d4c7fb.png)

vs.

![Screen Shot 2022-09-26 at 4 24 44 PM](https://user-images.githubusercontent.com/35355575/192383470-d8454bf1-a93e-4494-af27-0fd5320cd9ac.png)